pkg/ocicni/ocicni.go

```diff
@@ -616,7 +616,7 @@ func (plugin *cniNetworkPlugin) getCachedNetworkInfo(containerID string) ([]NetA
 		cacheFile := filepath.Join(dirPath, fname)
 		bytes, err := ioutil.ReadFile(cacheFile)
 		if err != nil {
-			logrus.Warningf("failed to read CNI cache file %s: %v", cacheFile, err)
+			logrus.Errorf("failed to read CNI cache file %s: %v", cacheFile, err)
```

Comment:
Same question here about the change from "Warning" to "Error".

Reply:
fixed!

Reply:
Same here,